ground/src/main/java/com/google/android/ground/ui/home/HomeScreenFragment.kt

```diff
@@ -71,6 +73,7 @@ class HomeScreenFragment :
       ->
       onApplyWindowInsets(insets)
     }
+    OssLicensesMenuActivity.setActivityTitle(getString(R.string.view_licenses_title))
```

Comment:
Can we move this to `showLicenses` method to keep all related code together?

Reply:
Done.